resources/js/Pages/post/Edit.vue

```diff
@@ -0,0 +1,309 @@
+                                        v-show="form.challenge_related"
+                                    >
+                                    </vue-select>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="flex justify-between mt-6">
+                            <div class="flex">
+                                <jet-button
+                                    :class="{ 'opacity-25': form.processing }"
+                                    :disabled="form.processing"
+                                >
+                                    Speichern
+                                </jet-button>
+                                <jet-action-message :on="form.recentlySuccessful" class="ml-3">
+                                    Gespeichert.
+                                </jet-action-message>
+                                <inertia-link
+                                    :href="route('app.post.index')"
+                                    class="secondary-button mr-3"
+                                >
+                                    Abbrechen
+                                </inertia-link>
+                            </div>
+
+                            <button
+                                @click="deletePost"
+                                type="button"
+                                class="danger-button"
+                            >
+                                Löschen
+                            </button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+    </app-layout>
+</template>
+
+<script>
+import JetActionMessage from "@/Jetstream/ActionMessage";
+import JetButton from "@/Jetstream/Button";
+import JetFormSection from "@/Jetstream/FormSection";
+import BannerPill from "@/components/BannerPill";
+import CategoryIcon from "@/components/CategoryIcon";
+import RadioInput from "@/input/RadioInput";
+import VueSelect from 'vue-next-select'
+import AppLayout from "@/Layouts/AppLayout";
+
+export default {
+    components: {
+        AppLayout,
+        CategoryIcon,
+        BannerPill,
+        JetActionMessage,
+        JetButton,
+        JetFormSection,
+        RadioInput,
+        VueSelect,
+    },
+    props: {
+        post: {
+            type: Object,
+        },
+        banners: {
+            type: Object,
+        },
+        challenges: {
+            type: Object,
+        },
+    },
+    data() {
+        const team = this.post.team
+        const currentChallengeId = team.current_challenges.length > 0 ? team.current_challenges[0].id : null
+        return {
+            form: this.$inertia.form({
+                subject: this.post.subject,
+                content: this.post.content,
+                banner_id: this.post.banner_id,
+                challenge_id: parseInt(this.post.challenge_id ? this.post.challenge_id : currentChallengeId),
```

Comment:
When I try to edit a post which does not have a challenge and the team does not have a challenge as well, this raises an exception (and the page is blank).

I think this is because `parseIn(null) => NaN`.

Making those changes fixes the bug:
```js
const currentChallengeId = team.current_challenges.length > 0 ? parseInt(team.current_challenges[0].id) : null
```
```suggestion
                challenge_id: this.post.challenge_id ? parseInt(this.post.challenge_id) : currentChallengeId,
```